### ISSUE TITLE
Launch button loads details of user previously interviewed

### ISSUE THREAD

**Issue by bkimondiu** [onaio/steps-app#216]
When a user bypasses the correct workflow of accessing a form i.e. instead of loading the form by clicking **Interview now**, they load the form directly from ODK Collect by selecting Fill Blank Form, clicking the Launch button loads the information of the user last interviewed. The Launch button loads nothing if no previous interview has been carried out.

Steps to replicate:
1. Download a form to ODK Collect
2. Add a household with members and do the selection
3. Click Interview now to carry out a survey
4. Add another household with members and do the selection
5. Instead of clicking Interview now, go directly to ODK Collect and select Fill Blank Form and select the survey form in use.
6. Click Launch

**Comment by ukanga** [comment]
Is there something missing from this explanation? Is the request that if someone goes to ODK Collect and picks fill the blank form, they expect the previous member information to be present?

**Comment by bkimondiu** [comment]
@uka No. The request is that no member information should be pulled when the form is accessed via ODK Collect instead of through the STEPS apk via the Interview now button.
